data/links/dslink-dart-digital-ocean.json

```diff
@@ -3,7 +3,8 @@
   "displayName": "Digital Ocean",
   "type": "Dart",
   "automated": {
-    "repository": "https://github.com/IOT-DSA/dslink-dart-digital-ocean.git"
+    "repository": "https://github.com/IOT-DSA/dslink-dart-digital-ocean.git",
+    "ignoreBuildScript": true
```

Comment:
These changes are because the build scripts in these links are pre-shipment of dependencies, and they break the Link Builder that now ships the packages and all their assets.

These build scripts usually copy the assets such as HTML to serve over an HTTP Server into their build directory. As this is no longer necessary since we ship the entire content, we will ignore these. This is somewhat temporary until we can remove these build scripts.